### ISSUE TITLE
Need consulting for subscription. Currently use unlimited chat.

### ISSUE THREAD

**Issue by marcozehner** [Writer-Sonic/AI-writer#9]
Is there an overview across you app and subscription levels?
The current subscription is valid until July - is there a way to upgrade properly?

**Comment by aryan1107** [comment]
Hi please use app to change payment plan as it's managed by apple you can change plan within the app. Before confirming apple will show you information of about current plan if it's active

**Comment by marcozehner** [comment]
Thank you for your reply. What I wanted to know: What are the differences between those subscriptions:Am I losing some benefits switching from „unlimited chat“ to „pro plan with image generation“ - and what would „unlimited word credits“ and „500000 word credits“ offer in comparison?Am 22.02.2024 um 06:29 schrieb Aryan Chaurasia ***@***.***>:﻿
Hi please use app to change payment plan as it's managed by apple you can change plan within the app. Before confirming apple will show you information of about current plan if it's active

**Comment by aryan1107** [comment]
Both have unlimited Ai chat expensive one also has image generation. It's expensive because image generation costs lots of GPU. Other than that everything same.

**Comment by marcozehner** [comment]
Dear Aryan,I subscribed yesterday to the image containing plan, but I am not satisfied with image generation: basically I only get photorealistic pictures in alsays the same style.On the other hand I checked Your imageGen AI Image & Art, that was quite sufficient from the beginning. Is there a way to combine these products in one subscription or did I miss features in Chat Unlimited & Ask Brutus AI app?Thx and best regards MarcoAm 26.02.2024 um 11:47 schrieb Aryan Chaurasia ***@***.***>:﻿
Both have unlimited Ai chat expensive one also has image generation. It's expensive because image generation costs lots of GPU. Other than that everything same.
